Ninjathulhu/Ninjathulhu.Game/EntityProperties.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Ninjathulhu.Game
+{
+    public class EntityProperties
+    {
+        public readonly ComponentSet FromPrefab;
+        public readonly ComponentSet FromSpawn;
+
+        public EntityProperties(ComponentSet prefabProperties, ComponentSet spawnProperties)
+        {
+            FromPrefab = prefabProperties;
+            FromSpawn = spawnProperties;
+        }
+
+        public object Get<TComponent>(string propertyName, object defaultValue = null)
+        {
+            object result;
+
+            result = FromSpawn.GetProperty(typeof(TComponent), propertyName);
```

Comment:
Might make sense to merge these into a single ComponentSet in the constructor, cloning `prefabProperties` and then insert-or-overwriting everything in `spawnProperties`. Is there a case where we'd want to remove an element from `spawnProperties` and unmask something in `prefabProperties`?